app/views/docs/rest.phtml

```diff
@@ -1,12 +1,337 @@
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-Response-Format: [VERSION-NUMBER]</code></td>
+        <td>Optional, used for backwards compatibility. The response will be formatted to be compatible the provided version number.</td>
+    </tr>
+    <tr>
+        <td><code>X-Fallback-Cookies: [FALLBACK-COOKIES]</code></td>
+        <td>Optional, used in scenarios where browsers do not allow third-party cookies. Often used when there is no <a href="/docs/custom-domains">Custom Domain</a>.</td>
+    </tr>
+    </tbody>
+</table>
 
-<h2><a href="/docs/realtime#authentication" id="authentication">Authentication</a></h2>
-<p>The REST API authenticates using Appwrite accounts and sessions. Both accounts and sessions can be created with REST using the <b>POST</b> <b>/v1/account</b>, <b>/v1/account/sessions/email</b>, <b>/v1/account/sessions/anonymous</b>, or <b>/v1/account/sessions/phone</b> endpoints.</p>
+<h3><a href="/docs/rest#no-headers" id="no-headers">Using Appwrite Without Headers</a></h3>
+<p>Some use cases do not allow the use of custom headers, such as when embedding images from Appwrite in HTML. In these cases, you can provide the Appwrite project ID using the query parameter <code>project</code>.</p>
+<div class="ide margin-bottom" data-lang="http" data-lang-label="HTTP">
+    <pre class="line-numbers"><code class="prism language-http" data-prism><img src="https://[HOSTNAME_OR_IP]/v1/storage/buckets/[BUCKET_ID]/files/[FILE_ID]/preview?project=[PROJECT_ID]"></code></pre>
+</div>
 
-<p>More information and examples of authenticating users can be found in the dedicated <a href="/docs/authentication">authentication guide</a>.</p>
+<h3>Client Authentication</h3>
+<p>You can create account sessions with POST requests to the <a href="docs/client/account">Account API</a>. Sessions are persisted using cookies like any other REST API. You can learn more about session persistence in the <a href="/docs/authentication#persistence">Authentication Guide</a>.</p>
+<p>The example below shows creating an account session with the <a href="/docs/client/account#accountCreateEmailSession">Create Account Session with Email</a> endpoint.</p>
+<div class="ide margin-bottom" data-lang="http" data-lang-label="HTTP">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>POST /v1/account/sessions/email  HTTP/1.1
+Host: [HOSTNAME_OR_IP]
+Content-Type: application/json
+X-Appwrite-Project: 5df5acd0d48c2
+X-Appwrite-Response-Format: 1.0.0
+
+{"email":"example@email.com","password":"password"}</code></pre>
+</div>
+
+<h2>Server Authentication</h2>
+<p>Server integrations use API keys to authenticate and are typically used for backend appications.</p>
+<p>You don't need to create an account session before making requests to Server APIs. Simply pass an <a href="/docs/keys">API key</a> in the <code>X-Appwrite-key: [API-KEY]</code> header with the appropriate scopes.</p>
```

Comment:
> You don't need to create an account session before making requests to Server APIs.

Consider re-wording this; you can make requests to almost all routes using a key, not just server specific APIs